include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -358,22 +358,59 @@ struct buffered_piece_collection
     // Check if a turn is inside any of the originals
     inline void check_turn_in_original()
     {
+        auto const& strategy = m_strategy;
+
+        struct specific_options : experimental::partition_options
+        {
+            struct include_turn_policy
+            {
+                static inline bool apply(buffer_turn_info_type const& turn)
+                {
+                    return turn.is_turn_traversable && ! turn.within_original;
+                }
+            };
```

Comment:
Doing this, I noticed that we sometimes use the `include_turn_policy` but sometimes have it in the get_box/overlaps functors. For that reason, if this functionality is always there, we could also choose to get rid of these options.
(the one left is the default partition size but we can specify it as an optional parameter).
What do you think?

Reply:
I omitted it, it was really just about expanding the bounding box.